zenodo-json-schema.json

```
@@ -609,8 +609,7 @@
           },
           "orcid": {
             "description": "ORCID identifier for creator.",
-            "type": "string",
-            "pattern": "^[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{4}$"
+            "type": "string"
```

actually only the last char can be X, so,
```suggestion
            "type": "string",
            "pattern": "^[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]$"
```

---

is that better than [0-9A-Z]?

---

> is that better than [0-9A-Z]?

a bit stricter: "The last character in the ORCID iD is a checksum. In accordance with ISO/IEC 7064:2003, MOD 11-2, this checksum must be "0" - "9" or "X", a capital letter X which represents the value 10"

---

well discovered